_docs/concepts/security/mutual-tls.md

```diff
@@ -76,7 +76,7 @@ Service-to-service communication is tunneled through the client side [Envoy](htt
 
 Istio 0.2 supports services running on both Kubernetes pods and VM/bare-metal machines. We use different key provisioning mechanisms for each scenario.
 
-For services running on Kubernetes pods, the per-cluster Istio Citadel (acting as Certificate Authority) automates the key & certificate management process. It mainly performs four critical operations :
+For services running on Kubernetes pods, the per-cluster Citadel (acting as Certificate Authority) automates the key & certificate management process. It mainly performs four critical operations :
```

Comment:
Remove space between operations and the colon.

Reply:
Done.